packages/js/src/account/account-manager.ts

```diff
@@ -217,7 +216,7 @@ export abstract class AccountManager implements NearAccountManager {
   }
 
   protected get connection(): nearAPI.Connection {
-    return new nearAPI.Connection(this.networkId, this.provider, this.signer);
+    return new nearAPI.Connection(this.networkId, this.provider, this.signer, `jsvm.${this.networkId}`);
```

Comment:
What is this for?

Reply:
I used the incorrect near api js version cached on my mechine, so I add a new param to run test on my local mechine, I will changed it